Plugins/NativeXr/Source/NativeXr.cpp

```diff
@@ -1505,6 +1565,24 @@ namespace Babylon
                 }
             }
 
+            Napi::Value GetJointPose(const Napi::CallbackInfo& info)
+            {
+                assert(info[0].IsExternal());
+
```

Comment:
this will only throw in debug builds. Do we want to have this function return undefined if this check fails in release builds?

Reply:
Right now, the only data that's ever passed in is the jointSpace created above, on line 183. I talked with syntheticMagus about it, and decided to leave the assert in there in case a dev ever decides to implement JointSpace internally.